test/nuts/tracking/basics.nut.ts

```diff
@@ -133,6 +133,13 @@ describe('end-to-end-test for tracking with an org (single packageDir)', () => {
       );
     });
 
+    it('an empty "pull" retrieve is not an error', () => {
```

Comment:
this assertion moved to forceIgnore.nut and I forgot to remove it from basics.

[explanation: there's a profile change here.  I had to ignore profiles over in the forceIgnore.nut to ensure an empty "pull"